### PR TITLE
update core concept description

### PR DESCRIPTION
**Pull request by davidcarlisle**
as discussed on the call today

### CODE REVIEW

File: src/intent.html [hunk 1]
```diff
@@ -176,27 +176,55 @@ <h3 id="mixing_intent_dictionaries">Intent Concept Dictionaries</h3>
...
+       Group based on experience with different AT implementations and
+       following the guidelines set out in [[?Concept-Lists]].
+       
+
+       <p>Some readings benefit from annotating the kind of
+       mathematical object, rather than giving an explicit concept
+       name to be spoken.  For such cases, one should use an Intent
+       [=property=] instead of an Intent [=concept=].  The Math
+       Working Group maintain a similar list of <a
+       href="https://w3c.github.io/mathml-docs/intent-core-concepts/">list
```
Comment (dginev) [inline src/intent.html]: Is this intentionally the same link? Could we at least use a `#properties` placeholder for the part of the page that will hold the properties? So one of:

```
https://w3c.github.io/mathml-docs/intent-core-concepts/#properties
```
or, if meant to be a separate note
```
https://w3c.github.io/mathml-docs/intent-core-properties
```

File: src/intent.html [hunk 2]
```diff
@@ -176,27 +176,55 @@ <h3 id="mixing_intent_dictionaries">Intent Concept Dictionaries</h3>
...
+       of core property names</a> as described in the following
+       section.</p>
+
+       <p>AT reading MathML attributed with a concept (or property)
+       in these lists SHOULD consider this concept (or property) to be
+       a hint how the content could be read.  AT MAY use this concept
```
Comment (dginev) [inline src/intent.html]: `AT MAY` would be more legible as `AT systems MAY` here. 

It is easy to misread when the two separate all-caps terms touch.

File: src/intent.html [hunk 3]
```diff
@@ -176,27 +176,55 @@ <h3 id="mixing_intent_dictionaries">Intent Concept Dictionaries</h3>
...
+       as a hint to improve braille generation.  However, because
+       common notations have many specialized ways of being spoken AT
+       is NOT constrained to use the name given.  For example, AT may
+       vocalize <q>three quarters</q>, as <q>three over x</q> (for
```
Comment (dginev) [inline src/intent.html]: I believe the word "as" before `three over x` shouldn't be there.

**Review by dginev — APPROVED** [review]
Looks great in general - I left a few comments that may need minor edits.

File: src/intent.html (hunk 1 above)
Comment (davidcarlisle) [reply src/intent.html]: sorry I'll fix, it was supposed to be https://w3c.github.io/mathml-docs/intent-core-properties

File: src/intent.html (hunk 2 above)
Comment (davidcarlisle) [reply src/intent.html]: done

File: src/intent.html (hunk 3 above)
Comment (davidcarlisle) [reply src/intent.html]: I think it's a bit over abbreviated and hard to understand I'll adjust and ping you once commited,

**Comment by dginev** [comment]
Looks even better, thanks again.

**Pull request merged**